.github/workflows/test_pkg_gen.yaml

```diff
@@ -22,8 +22,7 @@ jobs:
         config:
           - {os: windows-latest, r: 'release', rust-version: 'stable-gnu',  rtools-version: '42'}
           - {os: windows-latest, r: 'release', rust-version: 'stable-msvc', rtools-version: '42'}
-          - {os: windows-latest, r: 'devel',   rust-version: 'stable-gnu',  rtools-version: '42'}
```

Comment:
It seems I forgot to remove the extra GNU runner on #219. Let me remove this in this pull reuqest.